public/common/TracyQueue.hpp

```diff
@@ -395,6 +398,23 @@ struct QueueMessageColorFatThread : public QueueMessageColorFat
     uint32_t thread;
 };
 
+struct QueueBlob
+{
+    int64_t time;
+};
+
+struct QueueBlobData : public QueueBlob
+{
+    uint64_t encoding;
```

Comment:
What is the meaning of the encoding values? The TracyBlob() macro requires setting a value. How will user know what to set there? If "the encoding parameter is meant to add some decoding ability in the profiler later on", how will this future functionality be able to work, when there is no "restricted for future use" set of values that shouldn't be used by users?

Reply:
I am not completely done with this. My (totally untested) idea was to have a facility for executing user supplied dissectors, like in Wireshark, maybe even Lua based. This facility could use the encoding value to choose the dissector.